helpers/theme.js

```diff
@@ -6,14 +6,25 @@ const {
 	HOST,
 } = require('../config/global');
 
+const instanceSpecificTitle = () => {
+	switch (SC_THEME) {
+		case 'n21':
+			return 'Niedersächsischen Bildungscloud';
```

Comment:
What is the value of SC_TITLE for n21?

Reply:
'Niedersächsische Bildungscloud' from an old registration E-Mail. I implemented instanceSpecificTitle to allow exceptions in 
any instance if needed (just add another case)

Reply:
Ive renamed the function to make its use more intuitive